proto/message_contents/message.proto

```diff
@@ -67,4 +67,38 @@ message DecodedMessage {
     string content_topic = 6;
     ConversationReference conversation = 7;
     bytes content_bytes = 8;  // encapsulates EncodedContent
+}
+
+// Plaintext header included with messages, visible to all
+message PadlockMessageHeader {
+    uint64 sent_ns = 1;
+    // The session_id is derived via SHA256. Any other identifier will leak the
+    // sender of the message.
+    string session_id = 2;
```

Comment:
What is the sessionId in this case --  the hash of the installationPublicKey?

Is there a reason why this data is unencrypted?

Reply:
The issue here is that then the session_id becomes an identifying piece of metadata, and would leak information about pseudonymous conversation activity. Which intern could Identity senders through simple analysis.

- search inbound topics for messages with the same session_id
- Find Association for a given installationId

If the intention is to meet our sender privacy goals then session_id needs to be considered private. If the intention is to address those in a fast follow, then the header could contain more information such as the actual sending user, the conversation_id, etc which would make a sealed sender implementation in the future easier.